backend/inspirehep/submissions/grobid_authors_parsers.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't know if submissions is the place for that, maybe place it in editor/utils?